src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,56 @@
+package duke;
+
+/**
+ * Represents a Personal Assistant Chatbot that helps a person keeps track of various things.
+ *
+ * @author Sherman Ng Wei Sheng
+ */
+public class Duke {
+    private final Storage storage;
+    private final Ui ui;
+    private TaskList list;
+
+    /**
+     * Constructor specifying the file path of the database to be stored and retrieved from for this bot.
+     *
+     * @param filePath File path to store and retrieve the duke bot information.
+     */
+    public Duke(String filePath) {
+        this.ui = new Ui();
+        this.storage = new Storage(filePath);
+        try {
+            this.list = new TaskList(storage.load());
+        } catch (DukeException e) {
+            this.ui.printMessage(e.getMessage());
+            this.list = new TaskList();
+        }
+    }
+    /**
+     * Starts the Chatbot, listens for user input and executes the command accordingly.
+     */
+    public void start() {
+        this.ui.printGreetings();
+
+        boolean isExit = false;
+        while (!isExit) {
+            try {
+                String input = this.ui.getInput();
+                Command c = Parser.parse(input);
```

Comment:
Command C can be renamed to reflect it better. Something like ParsedCommand would be more helpful to the readers